components/collections/src/codepointtrie/cptrie.rs

```diff
@@ -13,6 +13,7 @@ use core::num::TryFromIntError;
 use core::ops::RangeInclusive;
 use yoke::Yokeable;
 use zerofrom::ZeroFrom;
+use zerovec::ZeroSlice;
 use zerovec::ZeroVec;
 use zerovec::ZeroVecError;
 
```

Comment:
Request: Can you migrate CodePointMapBorrowed to use CodePointTrieBorrowed?

Reply:
Filed #2558 and #2557.